src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -313,7 +313,9 @@ private String getEmailTemplateName(SubscriptionType subscriptionType,
 
     private String getLetterTemplateName(SubscriptionType subscriptionType, NotificationEventType notificationEventType) {
         String letterTemplateName = notificationEventType.getId();
-        if (FALLBACK_LETTER_SUBSCRIPTION_TYPES.contains(notificationEventType)) {
+        if (subscriptionType != null
+            && (FALLBACK_LETTER_SUBSCRIPTION_TYPES.contains(notificationEventType)
```

Comment:
DWP_RESPONSE_RECEIVED_NOTIFICATION is a fallback letter, therefore it should be in the FALLBACK_LETTER_SUBSCRIPTION_TYPES list and the following condition is not required in this `if...statement`